### DISCUSSION TITLE
Compatible with other Launch Monitors?

### DISCUSSION THREAD

**Discussion by durandman** [springbok/MLM2PRO-GSPro-Connector#58] [Q&A] [answered]
Is this software compatible or able to be rendered to account for other launch monitors - specifically the Golfjoy GDS Plus? 

Golfjoy says this LM is available for connection to GSPro, just need a program to connect them.

Thanks!

**Comment by wonder99** [comment, marked as answer]
It looks doable, just as we added Mevo+.  We'd need a bunch of screenshots of the metrics from your screen-mirrored application.  And we'd need you to help with the final testing of course.  I haven't heard of that one

**Reply by durandman** [reply]
Awesome! So I'm clear, the screen mirrored app's function is to take screen shots of current software to communicate that to connector and GSPro?

Is there a way for LM to send data directly to this connector app?

**Comment by wonder99** [comment]
On Wed, Dec 20, 2023 at 3:09 PM durandman ***@***.***> wrote:

> Awesome! So I'm clear, the screen mirrored app's function is to take
> screen shots of current software to communicate that to connector and GSPro?
>
That's right.  Exputt putting works the same way.


> Is there a way for LM to send data directly to this connector app?
>
No, the connector operates on screenshots.  It keeps taking them and looks
for changes.  When it sees a big change, it focuses on the 6 metrics, turns
the snapshots into digits, and sends them to GSPRO using their OpenAPI.
We have found that launch monitor companies don't give out the details
needed to send data directly from their device or app to us.  Also, using
screenshots makes all monitors pretty much the same, rather than some
custom communication for each one.

- Paul

**Reply by durandman** [reply]
Great info, thanks!

I'll work on sending screenshots. Can I send those directly/privately?

**Comment by wonder99** [comment]
Ya, you can send them directly to my email account, ***@***.***

**Reply by durandman** [reply]
haha. All jokes aside - where should I share? On here? Please be kind as I'm new to github.

**Comment by wonder99** [comment]
Haha no, that's my real email address.   You can use it.
Github is too cumbersome.

**Reply by durandman** [reply]
okay... maybe its a viewing issue on my end, but this is all I see. 
![1](https://github.com/springbok/MLM2PRO-GSPro-Connector/assets/154464298/02b82a96-36d6-4d3a-ba39-9459c024f45f)

**Comment by wonder99** [comment]
Oh that's funny github blocked it. Can you reach me on discord or Facebook?

**Reply by durandman** [reply]
Yeah FB would be great! 

www.facebook.com/paul.durand.3745

**Comment by akselhetland** [comment]
will this connector work with the Fullswing Kit LM?

**Reply by wonder99** [reply]
Yes, support has been added

**Comment by GolfingAlot** [comment]
WIll this connect work with the VoiceCassie SC4?

**Reply by wonder99** [reply]
Yes, it should

**Comment by springbok** [comment]
I’m currently working through a few issues with two other sc4 owners, will let you know when an update is released.

**Reply by fergus040** [reply]
I am happy to test or provide more screenshots if needed.  Looking forward to trying this out

**Reply by Zack1972-stack** [reply]
Hi,wil it work on sc4 pro?

**Reply by fergus040** [reply]
My testing last night was inconsistent.  I would say too inconsistent for play.  Two issues, new data did not always read and some issues with 6 vs 5 identification.  From the message above it's seems like the issue is known.  I will trying playing around with android font sizing to see if it helps.

**Comment by GolfingAlot** [comment]
Thank You.
W!nston
"Inside me is a weak heart but behind it is a strong God"
"Life isn't about waiting for the storm to pass....It's about learning to dance in the rain"
"When you feel you're "drowning" in life's situations, don't worry, Your LifeGuard walks on water !"

Sent with [Proton Mail](https://proton.me/mail/home) secure email.

**Reply by springbok** [reply]
Here is pre-release version you can try.

https://www.dropbox.com/scl/fi/9m9e3ze4rw4pqmo01lq2e/MLM2Pro-GSPro-Connector_V1.04.14.zip?rlkey=pizxrfowmkbmwhs9m718haxqs&st=ag4v7y97&dl=0

**Comment by GolfingAlot** [comment]
Thank you!

**Comment by tonyayna** [comment]
I'm not sure is it possible to connect with another putt device - https://oneputt.me/en. We are the developer of the putting device. Not sure how to discuss on bluetooth characteristic and value that the device send out?

**Reply by springbok** [reply]
To clarify, your device has a Bluetooth interface? If so we can add it as a new putting device and interface to it via BT, but we will need documentation detailing the characteristics & values it sends out.

**Reply by tonyayna** [reply]
Dear, 
We did the code update to connect to our device as per link below. 

https://github.com/tonyayna/MLM2PRO-GSPro-Connector-ONEPUTT/

However, as some of our customer try to connect to MLM2Pro, it seem not working as it's required 'MLM2PROSecret'. Not sure can you add the code to main branch and create a release. Thank you.

**Comment by wonder99** [comment]
It is certainly possible, it just needs some development.  Do you have documentation for its interface?

**Reply by tonythk** [reply]
> It is certainly possible, it just needs some development.  Do you have documentation for its interface?

Great. Let me prepare the documentation and share it soon

**Comment by wonder99** [comment]
I created an account on Line to see what its price might be, but I don't see it.  Are these two apps the only way you plan to sell this device?  They don't instill a lot of confidence in me, since I've never heard of either of them.

**Reply by tonythk** [reply]
The product now is selling only in Thailand but we have a plan to export by this year. The price would be 439$ for US market.

**Reply by tonyayna** [reply]
Pls check the ble document at
https://oneputt.me/media/oneputt_ble_documentation_v1.0.pdf

**Comment by springbok** [comment]
I can email it to you if you send me your email address, it’s not publicly available as we don’t want to upset Rapsodo by publishing this information.

**Reply by tonyayna** [reply]
Thank you so much. Please email to me at tony@progolf.asia

**Comment by runpuddrun** [comment]
Is it possible to add protee vx support?
